src/utils/types.js

```diff
@@ -114,9 +114,15 @@ export function asStringParamWhen (param, value, predicate) {
 export function mapAsParams (paramsMap) {
   if (paramsMap && isObject(paramsMap)) {
     const array = []
-    Object.keys(paramsMap).forEach((key) => {
+    Object.entries(paramsMap).forEach(([key, values]) => {
       const value = paramsMap[key]
-      value && !isObject(value) && value.length && array.push([encodeURIComponent(key), encodeURIComponent(value)])
+      if (value && !isObject(value) && value.length) {
+        if (Array.isArray(values)) {
+          values.forEach(id => array.push([encodeURIComponent(key), encodeURIComponent(id)]))
+        } else {
+          array.push([encodeURIComponent(key), encodeURIComponent(value)])
+        }
+      }
```

Comment:
This is a bit inconsistent: you use a combination of `entries` with `paramsMap[key]`. One of these should be enough. 

Also, using `entries` instead of `keys` increased the size of the generated tag even though the functionality barely changed. I propose to continue using `keys`:

```
export function mapAsParams (paramsMap) {
  if (paramsMap && isObject(paramsMap)) {
    const array = []
    Object.keys(paramsMap).forEach((key) => {
      const value = paramsMap[key]
      if (value && !isObject(value) && value.length) {
        if (isArray(value)) {
          value.forEach(entry => array.push([encodeURIComponent(key), encodeURIComponent(entry)]))
        } else {
          array.push([encodeURIComponent(key), encodeURIComponent(value)])
        }
      }
    })
    return array
  } else {
    return []
  }
}
```

Reply:
you're right, it looks better that way, adjusted the code! thank you!